packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -1186,22 +1178,18 @@ Terra.describeViewports('DateTimePicker', ['large'], () => {
           expect(minuteInRange).toBe(true);
         });
 
-        it('should set date to today and now plus 1 minute in the second input', () => {
+        it('should set date to today and now plus 1 seconds in the second input', () => {
           browser.url('/raw/tests/cerner-terra-framework-docs/date-time-picker/date-time-picker-with-seconds');
           browser.refresh();
           $('input[name="terra-time-second-input"]').click();
           browser.keys('+');
 
-          const today = moment.utc().add(1, 'minutes');
+          const today = moment.utc().add(1, 'seconds');
           expect($('input[name="terra-date-year-input"]')).toHaveValue(today.year().toString());
           expect($('input[name="terra-date-month-input"]')).toHaveValue((`0${(today.month() + 1)}`).slice(-2));
           expect($('input[name="terra-date-day-input"]')).toHaveValue((`0${today.date()}`).slice(-2));
           expect($('input[name="terra-time-hour-input"]')).toHaveValue((`0${today.hour()}`).slice(-2));
           expect($('input[name="terra-time-minute-input"]')).toHaveValue((`0${today.minute()}`).slice(-2));
-          const secondsValue = Number($('input[name="terra-time-second-input"]').getValue());
-          const secondsTimeValue = today.seconds();
-          const secondsInRange = (secondsTimeValue === secondsValue + 1 || secondsTimeValue === secondsValue);
-          expect(secondsInRange).toBe(true);
```

Comment:
This test says that you are testing +1 sec to the seconds but I cannot see the validation for seconds field here.

Reply:
Updated